packages/agent/src/WrtnAgent.ts

```diff
@@ -227,6 +197,65 @@ export class WrtnAgent {
     return this.token_usage_;
   }
 
+  /**
+   * @internal
```

Comment:
Is there a reason why you designated it as internal?

Reply:
No special reason

Reply:
To hide the function in the declaration file